src/explorer.ts

```diff
@@ -264,8 +267,25 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
         return kubefsUri(this.namespace, this.resourceId, outputFormat);
     }
 
-    getChildren(kubectl: Kubectl, host: Host): vscode.ProviderResult<KubernetesObject[]> {
-        return [];
+    async getChildren(kubectl: Kubectl, host: Host): Promise<KubernetesObject[]> {
+        if (this.kind !== kuberesources.allKinds.pod) {
+            return [];
+        }
+        const result = await kubectl.asJson<Pod>(`get pods ${this.metadata.name} -o json`);
+        if (result.succeeded) {
+            const pod = result.result;
+            let ready = 0;
+            pod.status.containerStatuses.forEach((status) => {
+                if (status.ready) {
+                    ready++;
+                }
+            });
+            return [
```

Comment:
I'm a bit doubtful about using child nodes to show properties of a resource - that's not an idiom we've used before, and it feels a bit odd.  A possible alternative would be to show this info on the tooltip, but that's problematic because I think we need to construct the tooltip synchronously.  Or a 'summary status' command perhaps.

Reply:
moved to a different PR.